data/transition-sites/esfa_ncs.yml

```
@@ -0,0 +1,8 @@
+---
+site: esfa_ncs
+whitehall_slug: education-and-skills-funding-agency
+homepage: http://www.nationalcareers.service.gov.uk
+tna_timestamp: 20201010101010 # Stub timestamp - site not in TNA
```

```suggestion
tna_timestamp: 20190309013456
```

I found the latest archive of this site with that date: https://webarchive.nationalarchives.gov.uk/20190309013456/https://nationalcareersservice.direct.gov.uk/